### PR TITLE
Temporarily disable cache-machine (bug 881405)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=881405

This is an experiment to see what's the impact of disabling cache-machine on Zamboni. If it works, we can remove it properly.

(Haven't ran the tests yet, I'm curious to see if they pass...)

### CODE REVIEW

File: mkt/translations/tests/testapp/models.py [hunk 1]
```diff
@@ -1,30 +1,46 @@
...
+
+class ManagerWithTranslations(models.Manager):
+    def get_query_set(self):
+        qs = TransformQuerySet(self.model)
+        if hasattr(self.model._meta, 'translated_fields'):
+            qs = qs.transform(transformer.get_trans)
+        return qs
+
+
+class TranslatedModel(models.Model):
```
Comment (robhudson) [inline mkt/translations/tests/testapp/models.py]: Do we care about the `created` and `modified` fields that were on `ModelBase`?

File: mkt/translations/tests/testapp/models.py [hunk 2]
```diff
@@ -1,30 +1,46 @@
...
     no_locale = TranslatedField(require_locale=False)
 
+    objects = ManagerWithTranslations()
+
 models.signals.pre_save.connect(save_signal, sender=TranslatedModel,
                                 dispatch_uid='testapp_translatedmodel')
 
 
-class UntranslatedModel(ModelBase):
+class UntranslatedModel(models.Model):
```
Comment (robhudson) [inline mkt/translations/tests/testapp/models.py]: Noticed this isn't used in our codebase.

File: mkt/translations/tests/testapp/models.py [hunk 3]
```diff
@@ -1,30 +1,46 @@
...
     """Make sure nothing is broken when a model doesn't have translations."""
     number = models.IntegerField()
 
+    objects = ManagerWithTranslations()
+
 
-class FancyModel(ModelBase):
+class FancyModel(models.Model):
```
Comment (robhudson) [inline mkt/translations/tests/testapp/models.py]: Neither is this.

File: mkt/translations/tests/testapp/models.py (hunk 1 above)
Comment (diox) [inline mkt/translations/tests/testapp/models.py]: Since it's just a test app, no.

File: mkt/translations/tests/testapp/models.py (hunk 3 above)
Comment (diox) [inline mkt/translations/tests/testapp/models.py]: This and above are just test apps used in `mkt/translations/tests/test_models.py`

File: mkt/translations/tests/testapp/models.py (hunk 2 above)
Comment (diox) [inline mkt/translations/tests/testapp/models.py]: See below.

File: mkt/translations/tests/testapp/models.py (hunk 1 above)
Comment (robhudson) [inline mkt/translations/tests/testapp/models.py]: Heh, missed that.

**Comment by diox** [comment]
Setting added (set to `True` only on dev/stage) and tests fixed.

File: manage.py [hunk 4]
```diff
@@ -6,10 +6,11 @@
...
 
-
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'mkt.settings')
-if len(sys.argv) > 1 and sys.argv[1] == 'test':
-    os.environ['DJANGO_SETTINGS_MODULE'] = 'settings_test'
+if not 'DJANGO_SETTINGS_MODULE' in os.environ:
+    if len(sys.argv) > 1 and sys.argv[1] == 'test':
+        os.environ['DJANGO_SETTINGS_MODULE'] = 'settings_test'
+    else:
+        os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'mkt.settings')
```
Comment (robhudson) [inline manage.py]: :+1:

**Comment by andymckay** [comment]
r+wc

**Pull request merged**